kbdd/src/test/kotlin/ru/fix/kbdd/asserts/KPathTest.kt

```diff
@@ -123,4 +123,24 @@ internal class KPathTest {
                 .single { it["name"].isEquals("wallet") }["fields"]
                 .size().isEquals(2)
     }
+
+    @Test
+    fun `assert different type numbers`() {
+        val data = mutableMapOf<String, Any?>("one" to 1, "two" to 2.0)
+
+//        Int with Int
+        KPath(data)["one"].isEquals(1)
+        KPath(data)["one"].isLessThan(2)
+
+//        Int with Long
+        KPath(data)["one"].isEquals(1L)
+        KPath(data)["one"].isLessThan(2L)
+
+//        Int with Double
+        KPath(data)["one"].isEquals(1.0)
```

Comment:
Double and Floats should not be examine with exact match operators   
Could you please add methods with delta for double and float values
```
isLessThan(other: Double, delta: Double = 0.1) 
isEquals(other: Double, delta: Double = 0.1) 
...
```